tests/backend/core/Core/Functional/res/customerConfig1.yaml

```diff
@@ -0,0 +1,2 @@
+name: 'Foobar'
```

Comment:
shouldn't this be an array of customers?

Reply:
Not that I know of. As far as I understood the action to create a customer is very rare, so I assumed the config file would be created for that circumstance and deleted afterwards.

I can refactor it to accept an array if you want.

Reply:
I see. This is a legit implementation, I just thought that this configuration file keeps all customers. This way it might be a bit more resilient and easier to deploy

Reply:
That seems to deviate from "create a customer via config" to "manage customers via config". E.g. if I would see such config for the first time and be told I can add customers to it and they will automatically be created, I'd assume that I can simply remove an entry in the config to remove a customer from the system.

Reply:
Indeed. At least it should not be allowed to be used any more (aka new subdomains_allowed)

Reply:
Considering the remaining time I suggest to stick to the current approach for now. We can still add improvements with followup stories.